test/hls_reference/video_mixer/g2QABXZpZGVv.m3u8

```diff
@@ -4,6 +4,8 @@
 #EXT-X-MEDIA-SEQUENCE:0
 #EXT-X-DISCONTINUITY-SEQUENCE:0
 #EXT-X-MAP:URI="video_header_g2QABXZpZGVv_part_0.mp4"
-#EXTINF:1.541666666,
```

Comment:
So before we had only one segment of 1.5s duration and now we have 2x, each of 5s? But we haven't changed the test file :thinking:

Reply:
I think this can happen because we add BlankVideoGenerator, so we will have a black screen before a proper multimedia.